.github/workflows/ecosystem.yml

```diff
@@ -0,0 +1,540 @@
+
+    - name: Configure Psi4, LLVM clang-cl (W)
+      if: runner.os == 'Windows'
+      run: |
+        cmake \
+          -S . \
+          -B objdir \
+          -G Ninja \
+          -D CMAKE_BUILD_TYPE=Release \
+          -D CMAKE_INSTALL_PREFIX=./install \
+          -D CMAKE_C_COMPILER=clang-cl \
+          -D CMAKE_CXX_COMPILER=clang-cl \
+          -D CMAKE_PREFIX_PATH="${CONDA_PREFIX}" \
+          -D CMAKE_INSIST_FIND_PACKAGE_pybind11=ON \
+          -D CMAKE_INSIST_FIND_PACKAGE_gau2grid=ON \
+          -D CMAKE_INSIST_FIND_PACKAGE_Libint2=ON \
+          -D CMAKE_INSIST_FIND_PACKAGE_qcelemental=ON \
+          -D CMAKE_INSIST_FIND_PACKAGE_qcengine=ON \
+          -D CMAKE_INSIST_FIND_PACKAGE_Libxc=ON \
+          -D ENABLE_OPENMP=ON \
+          -D ENABLE_XHOST=OFF \
+          ${{ matrix.cfg.cmargs }}
+
+    - name: Compile & Install Psi4
+      run: cmake --build objdir --config Release --target install -j
+
+    - name: Linked dependencies, ldd (L)
+      if: runner.os == 'Linux'
+      run: ldd objdir/stage/lib/psi4/core.*.so
+
+    - name: Linked dependencies, otool (M)
+      if: runner.os == 'macOS'
+      run: otool -L objdir/stage/lib/psi4/core.*.so
+
+    - name: Linked dependencies, objdump (W)
+      if: runner.os == 'Windows'
+      shell: cmd /C CALL {0}
+      run: objdump.exe -p objdir\stage\lib\psi4\core.*.pyd | grep dll
+
+      # Notes: This is clearing away the whole build environment (to make explicit the build and
+      #   runtime dependencies and addons). Then it's creating a new runtime environment based on
+      #   psi4 and base channel dependencies and addons. This is created at the same environment
+      #   name (so same location on disk as the build env) for non-relocatability reasons. Also
+      #   creating another runtime environment based on conda-forge channel addons. This is held in
+      #   reserve for later.
+    - name: Runtime Conda environment(s)
+      run: |
+        echo "::group::Prepare Environment"
+        conda deactivate
+        conda remove -n baseenv --all
+        conda env create --file run.yaml
+        if [[ "${{ matrix.cfg.base-channel }}" != "conda-forge" ]]; then
+        conda env create --file aux.yaml
+        fi
+        conda activate baseenv
+        echo "::endgroup::"
+        conda info
+        conda list -n baseenv
+        if [[ "${{ matrix.cfg.base-channel }}" != "conda-forge" ]]; then
+        conda list -n auxenv
+        fi
+
+    - name: Test OpenMP
+      working-directory: ./install/lib
+      run: python -c "from psi4 import core; core.set_num_threads(42); assert core.get_num_threads() == 42"
+
+    - name: Test Run, Spot, Run
+      working-directory: ./objdir
+      run: |
+        export PATH=../install/bin:$PATH
+        #export KMP_DUPLICATE_LIB_OK=TRUE
+        psi4 ../tests/tu1-h2o-energy/input.dat -o stdout
+
+      # step works but is redundant with next two
+    - name: Test Addons with CTest
+      if: false
```

Comment:
If this never runs, why is this part of the file here?

Reply:
ready-to-go when debugging needed. either the case of debugging the pytest environment (not as hard-coded as ctest) or debugging addon tests that also can run under ctest.

Reply:
A comment to the effect of "set true for debug" would be helpful.